internal/processor/filterset/factory/factory.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package factory
+
+import (
+	"fmt"
+
+	"github.com/open-telemetry/opentelemetry-collector/internal/processor/filterset"
+	"github.com/open-telemetry/opentelemetry-collector/internal/processor/filterset/regexp"
+	"github.com/open-telemetry/opentelemetry-collector/internal/processor/filterset/strict"
+)
+
+// Factory can be used to create FilterSets.
+type Factory struct{}
```

Comment:
Do we need a factory type? It does not have any members so it appears we could be served equality well by free standing functions `CreateFilterSet`, etc.

Reply:
Removed